plugins/org.eclipse.papyrus.uml.diagram.sequence.contribution/plugin.xml

```diff
@@ -103,5 +103,10 @@
        </Priority>
     </editpolicyProvider>
  </extension>
+ <extension
+       name="Lightweight Sequence Diagram" 
+       point="org.eclipse.papyrus.infra.properties.contexts">
+    <context appliedByDefault="true" contextModel="context/lightweight-seqd.contexts" isCustomizable="true"/>
```

Comment:
Should we embed your suggestion about integration with Papyrus, not to apply by default, as a comment here?

Reply:
Yes, I'll add a comment.